client/verta/tests/deployable_entity/test_artifacts.py

```diff
@@ -679,3 +681,33 @@ def test_setup_script(self, deployable_entity):
         assert deployable_entity.get_artifact(
             "setup_script"
         ).read() == six.ensure_binary(new_setup_script)
+
+
+class TestSetSchema:
+    def test_set_schema(self, deployable_entity):
+        input_schema = InputClass.schema()
+        output_schema = OutputClass.schema()
+        deployable_entity.set_schema(input=input_schema, output=output_schema)
+
+        assert deployable_entity.get_schema() == {
+            "input": input_schema,
+            "output": output_schema,
+        }
+
+    def test_set_schema_no_output(self, deployable_entity):
+        input_schema = InputClass.schema()
+        deployable_entity.set_schema(input=input_schema)
+
+        assert deployable_entity.get_schema() == {
+            "input": input_schema,
+        }
+
+    def test_set_schema_no_input_which_is_bad(self, deployable_entity):
```

Comment:
This is how I've been naming other tests that check for an error to be raised.

```suggestion
    def test_set_schema_no_input_error(self, deployable_entity):
```